scripts/zones/Mhaura/npcs/Ekokoko.lua

```diff
@@ -11,6 +11,8 @@ entity.onTrade = function(player, npc, trade)
 end
 
 entity.onTrigger = function(player, npc)
+    -- TODO: This is not a random sequence, but 51 is displayed on first interaction ever,
```

Comment:
Are you going to do this, or should we make an issue for this?

Reply:
Raised #1385